project/src/data/new-evaluations.js

```diff
@@ -0,0 +1,297 @@
+const getTextareas = function(){
+    return  {
+        items: [
+            {
+                title: "Should the candidate be hired?",
+                placeholder: "The type of project that is suitable for the candidate Is guidance required for the candidate"
+            },
+            {
+                title: "General Impression",
+                placeholder: "*required"
+            },
+            {
+                title: "Workflow, Leadership &amp; Soft Skills",
+                placeholder: "Describe the environment in which the candidate works. Describe any guidance or management experience"
+            }
+        ]
+    }
+}
+
+const getTechnicalLevels = function() {
+    return {
+        title: "Technical Level",
+        headers: ["Trainee", "Junior", "Middle", "Senior"],
+        columnData: [
+            {
+                inputLevels: ["T"]
+            },
+            {
+                inputLevels: ["J1","J2", "J3"]
+            },
+            {
+                inputLevels: ["M1", "M2", "M3"]
+            },
+            {
+                inputLevels: ["S1", "S2", "S3"]
+            }
+            
+        ]};
+}
+
+
+const getTechnicalAreas = function(){
+    const optionsItems = ["0", "1", "2", "3"];
+    const dropdownOptions1 = [
+        [
+            {
+                label: "Classes",
+                options: optionsItems
+            },
+            {
+                label: "Exception Handling",
+                options: optionsItems
+            },
+            {
+                label: "Version Control",
+                options: optionsItems
+            }
+        ],
+        [
+            {
+                label: "Access Modifiers",
+                options: optionsItems
+            },
+            {
+                label: "Design Patterns",
+                options: optionsItems
+            },
+            {
+                label: "Issue Tracking",
+                options: optionsItems
+            }
+        ],
+        [
+            {
+                label: "Polymorphism",
+                options: optionsItems
+            },
+            {
+                label: "RegEx",
+                options: optionsItems
+            }
+    ]
+    ];
+    const dropdownOptions2 = [
+        [
+            {
+                label: "Protocol",
+                options: optionsItems
+            },
+            {
+                label: "Response Codes",
+                options: optionsItems
+            },
+            {
+                label: "Rest",
+                options: optionsItems
+            }
+        ],
+        [
+            {
+                label: "Headers",
+                options: optionsItems
+            },
+            {
+                label: "Request Methods",
+                options: optionsItems
+            },
+            {
+                label: "Sessions & Cookies",
```

Comment:
Do you properly escape these properties before inserting them into the DOM? If not, I'm gonna say it: UNESCAPED HTML ENTITY! :stuck_out_tongue:

Reply:
I do now :))